static/templates/left_sidebar.hbs

```diff
@@ -101,6 +101,13 @@
                 {{#unless is_guest }}
                     <div id="subscribe-to-more-streams"></div>
                 {{/unless}}
+                <div id="login-link-container" class="only-visible-for-spectators">
+                    <a id="login-link">
```

Comment:
Let's use `login_button` class directly here.

Reply:
Roger that!